.storybook/main.ts

```diff
@@ -1,4 +1,4 @@
-import viteConfig from "../dev/vite.config";
```

Comment:
I moved some vite configs around. It always felt weird that we used `dev/vite.config.ts` as our base.  Now we have a shared base config in the root of the repo and both Storybook and `dev/` leverage it. 

This will pave the way in the future for us to use Vitest for testing (faster!!)